app/javascript/components/Footer/Footer.tsx

```diff
@@ -1,12 +1,64 @@
-import * as React from 'react';
+import React, { useState, useEffect } from 'react';
+import { Footer as UswdsFooter, GridContainer, FooterNav, Logo, Address } from '@trussworks/react-uswds';
 
+//import logoImg from 'uswds/src/img/logo-img.png';
 interface FooterProps {
 }
 
 export const Footer = (props: FooterProps) => {
+
+  const returnToTop = (
+    <GridContainer className="usa-footer__return-to-top">
+      <a href="#">Return to top</a>
+    </GridContainer>
+  )
+
   return (
     <div id="serp-footer-wrapper">
-
+      <UswdsFooter
+        size="slim"
+        returnToTop={returnToTop}
+        primary={
+          <div className="usa-footer__primary-container grid-row">
+            <div className="mobile-lg:grid-col-8">
+              <FooterNav
+                size="slim"
+                links={Array(4).fill(
```

Comment:
Should we move this to a `const`?

Reply:
@ddzz do you mean to move the links to a const?